### PR TITLE
(#489) Out-of-bounds page number displays blank screen

### PR DESCRIPTION
**Pull request by alerosad**
- added conditional to <div className="results-page__list">
- app displays no results page, with page number if given an invalid page number

### CODE REVIEW

**Comment by alerosad** [comment]
https://react-app-dev.cancer.gov/clinical-trials-search-app/pr-500

File: src/views/ResultsPage/ResultsPage.jsx [hunk 1]
```diff
@@ -556,6 +585,8 @@ const ResultsPage = () => {
 		);
 	};
 
+	console.log('pageState: ', pageState);
```
Comment (seyilonge-nci) [inline src/views/ResultsPage/ResultsPage.jsx:588]: Left over `console.log`

**Comment by blilianyu** [comment]
- Remove skeleton below heading
- Invalid is misspelled
Screen Shot 2022-04-12 at 3 00 31 PM

cc @mworrest

**Comment by alerosad** [comment]
https://react-app-dev.cancer.gov/clinical-trials-search-app/pr-500

**Comment by blilianyu** [comment]
Looks good! 
cc @mworrest

**Review by blilianyu — APPROVED** [review]

**Review by adriancofie — APPROVED** [review]

**Review by arcepaul — APPROVED** [review]

**Pull request merged**